diff -Naurp a/Lib/test/test_gdb.py b/Lib/test/test_gdb.py
--- a/Lib/test/test_gdb.py	2015-02-25 05:27:45.000000000 -0600
+++ b/Lib/test/test_gdb.py	2015-02-25 19:49:47.260238249 -0600
@@ -196,6 +196,10 @@ class DebuggerTests(unittest.TestCase):
             # Issue #19753: missing symbols on System Z
             'Missing separate debuginfo for ',
             'Try: zypper install -C ',
+            'warning: Unable to open',
+            'Missing separate debuginfo for',
+            'Try: yum --disablerepo=',
+            'Undefined set print command'
             )
         for line in errlines:
             if not line.startswith(ignore_patterns):
